test/acceptance/ApiSubscriptionFieldsSpec.scala

```diff
@@ -175,14 +174,13 @@ class ApiSubscriptionFieldsSpec extends AcceptanceTestSpec
       val sfr = contentAsJson(resultFuture).validate[SubscriptionFieldsResponse]
 
       sfr.isSuccess shouldBe true
-      sfr.get should matchPattern { case SubscriptionFieldsResponse(FakeRawIdentifier, _, SampleFields2) => }
-
+      sfr.get should matchPattern { case SubscriptionFieldsResponse(`fakeRawClientId`, "acontext", "1.0.2", _, SampleFields2) => }
     }
 
-    scenario("the API is called to DELETE a known subscription identifier") {
+    scenario("the API is called to DELETE a known subscription field") {
```

Comment:
`the API is called to DELETE existing subscription fields`

Reply:
Done